api/api.yaml

```diff
@@ -112,6 +94,31 @@ paths:
         '500':
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
 
+  /control/flows:
+    post:
+      tags: ['Control']
+      operationId: update_flow_state
+      description: >-
+        Updates the state of configuration resources on the traffic generator.
```

Comment:
this description is very generic. we should mention that it updates flow properties without disrupting transmit state.